omod/src/test/java/org/openmrs/module/mksreports/web/controller/MKSReportsManageControllerTest.java

```diff
@@ -67,7 +67,7 @@ public Properties getRuntimeProperties() {
 	
 	@Before
 	public void setup() throws Exception {
-		executeDataSet("testDataset.xml");
+		executeDataSet("ControllerTestDataset.xml");
```

Comment:
Being picky, but camelCase: **`controllerTestDataset.xml`**.
But why change the name? It's weird naming though, it doesn't really describe what's _in_ the test dataset.